perf/benchmark/security/generate_policies/generate_policies.go

```diff
@@ -0,0 +1,194 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/json"
+	"flag"
+	"fmt"
+
+	"sort"
+	"strings"
+
+	"github.com/ghodss/yaml"
+	"github.com/golang/protobuf/proto"
+
+	"istio.io/istio/pkg/util/protomarshal"
```

Comment:
I think we'd better not to import the "istio.io/istio" here, it's too large and that's why you're seeing the 300+ new files under the licenses directory from `make gen`.

And this is also a bit unreliable because the "istio.io/istio/pkg" is likely not supposed to be used externally and could change anytime for any reasons (like a code refactor in istio/istio), thus will likely cause some issues here.

Instead you can just copy the implementation here, it's small enough and quite independent to be copied here.